src/Tracing/TransactionContext.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Tracing;
+
+class TransactionContext extends SpanContext
+{
+    /**
+     * @var string|null Name of the transaction
+     */
+    public $name;
```

Comment:
Please make this property `private` and add the appropriate `getter` and `setter`

Reply:
For now we want to have simple objects carrying the properties.
Having getter/setters produces a lot more code, let's see we can still add this later if we really want to.

Reply:
This is a bad practice, plus in PHP we don't have typed properties before PHP 7.4 so there is nothing enforcing the value to be of the expected type. Also, if in the future we want to change the implementation of the class we cannot do it because everything is `public`